ELJSBridge/Source/Bridge.swift

```diff
@@ -72,17 +75,15 @@ public class Bridge: NSObject {
      */
     public func loadFromFile(filePath: String) throws {
         let filemanager = NSFileManager.defaultManager()
+        
         if filemanager.fileExistsAtPath(filePath) {
-            do {
-                let script = try String(contentsOfFile: filePath, encoding: NSUTF8StringEncoding)
-                do {
-                    try self.load(script)
-                } catch let catchError as NSError {
-                    throw catchError
-                }
-            } catch  {
-                throw NSError(ELJSBridgeError.FileDoesNotExist)
+            guard let script = try? String(contentsOfFile: filePath, encoding: NSUTF8StringEncoding) else {
```

Comment:
The original was being caught and propagated as `ELJSBridgeError.FileDoesNotExist`, but the way it was nested was confusing, so I moved that into the guard. Then I realized these were two different errors, so I created a new case.